go/logic/applier.go

```diff
@@ -300,7 +300,7 @@ func (this *Applier) AlterGhost() error {
 	return err
 }
 
-// AlterGhost applies `alter` statement on ghost table
+// AlterGhostAutoIncrement alter `AUTO_INCREMENT` clause on ghost table
```

Comment:
```suggestion
// AlterGhostAutoIncrement alters the `AUTO_INCREMENT` clause on ghost table
```
A suggestion in hindsight